# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/cors/redirect-userinfo.htm.ini
# Commit: ef6d81f0b298
# Full Hash: ef6d81f0b2989624a9918ec5d790dc2745db20ef
# Author: Valentin Gosu <valentin.gosu@gmail.com>
# Date: 2018-11-15 09:52:43
# Regressor Bug: 1495313
# File Overlap Count: 1
# Description:
#   Bug 1495313 - Allow password with empty username in URLs r=dragana
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D11254
# ==============================================================================

diff -r 06b12fd41ff1 -r ef6d81f0b298 testing/web-platform/meta/cors/redirect-userinfo.htm.ini
--- a/testing/web-platform/meta/cors/redirect-userinfo.htm.ini	Wed Nov 14 19:37:42 2018 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,4 +0,0 @@
-[redirect-userinfo.htm]
-  [Allow redirect without userinfo (:@ is trimmed during URL parsing)]
-    expected: FAIL
-